scripts/core/editor3/components/Editor3.jsx

```diff
@@ -0,0 +1,224 @@
+
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {Editor, EditorState, RichUtils, ContentState} from 'draft-js';
+
+export class Editor3 extends React.Component {
```

Comment:
All of this needs to be documented, module, classes, methods, properties. Please see the contribution guidelines (documentation section)(https://github.com/superdesk/superdesk-client-core/blob/master/CONTRIBUTING.md) for info how to do that.